apps/passport-client/src/state.ts

```diff
@@ -51,11 +51,11 @@ export interface AppState {
 
   // Dynamic (in-memory-only) state-machine for sync of E2EE encrypted data.
   uploadedUploadId?: string;
-  uploadingUploadId?: string;
   downloadedPCDs?: boolean;
-  downloadingPCDs?: boolean;
   loadedIssuedPCDs?: boolean;
   loadingIssuedPCDs?: boolean;
```

Comment:
did you intend to remove `loadingIssuedPCDs` as well? the fact that there's a global 'lock' on sync means we shouldn't need *any* of these 'in progress' fields, so it's confusing to me that you've removed two of them but not the third.

Reply:
See above.  loadingIssuedPCDs is now just a UI thing, not a "lock".